src/llmcompressor/datasets/utils.py

```diff
@@ -0,0 +1,36 @@
+from typing import Union
+
+from datasets import (
+    Dataset,
+    DatasetDict,
+    IterableDataset,
+    IterableDatasetDict,
+    load_dataset,
+)
+
+
+def get_raw_dataset(
```

Comment:
Why even maintain this function if it just transparently calls `load_dataset`?

Reply:
The imported `datasets.load_dataset` function can be our "central location" that loads the dataset. This function adds nothing.

Reply:
There should be a reason why it exists currently. Keeping that structure should be respected. 

Looks like this func had other roles and it was broken down to its current format.